README.md

```diff
@@ -105,17 +105,26 @@ The [documentation page](https://nixtla.github.io/neuralforecast/) contains all
 |[DeepAR](https://nixtlaverse.nixtla.io/neuralforecast/models.deepar.html)| RNN | recurrent ||✅|✅|✅|
 |[TCN](https://nixtlaverse.nixtla.io/neuralforecast/models.tcn.html)| CNN | recurrent |✅|✅|✅|✅|
 |[TimesNet](https://nixtlaverse.nixtla.io/neuralforecast/models.timesnet.html)| CNN | windows |✅|✅||✅|
+|[BiTCN](https://nixtlaverse.nixtla.io/neuralforecast/models.bitcn.html)| CNN | windows |✅|✅|✅|✅|
 |[DLinear](https://nixtlaverse.nixtla.io/neuralforecast/models.dlinear.html)| Linear | windows |✅|✅||✅|
+|[NLinear](https://nixtlaverse.nixtla.io/neuralforecast/models.nlinear.html)| Linear | windows |✅|✅||✅|
 |[MLP](https://nixtlaverse.nixtla.io/neuralforecast/models.mlp.html)| MLP | windows |✅|✅|✅|✅|
 |[NBEATS](https://nixtlaverse.nixtla.io/neuralforecast/models.nbeats.html)| MLP | windows |✅|✅||✅|
 |[NBEATSx](https://nixtlaverse.nixtla.io/neuralforecast/models.nbeatsx.html)| MLP | windows |✅|✅|✅|✅|
 |[NHITS](https://nixtlaverse.nixtla.io/neuralforecast/models.nhits.html)| MLP | windows |✅|✅|✅|✅|
+|[TiDE](https://nixtlaverse.nixtla.io/neuralforecast/models.tide.html)| MLP | windows |✅|✅|✅|✅|
+|[DeepNPTS](https://nixtlaverse.nixtla.io/neuralforecast/models.deepnpts.html)| MLP | windows |✅||✅|✅|
+|[TSMixer](https://nixtlaverse.nixtla.io/neuralforecast/models.tsmixer.html)| MLP | multivariate |✅|✅||✅|
+|[TSMixerx](https://nixtlaverse.nixtla.io/neuralforecast/models.tsmixerx.html)| MLP | multivariate |✅|✅|✅|✅|
+|[MLPMultivariate](https://nixtlaverse.nixtla.io/neuralforecast/models.mlpmultivariate.html)| MLP | multivariate |✅|✅|✅|✅|
```

Comment:
Multivariate models do not support probabilistic forecasting

Reply:
@elephaint Ok I'll fix this. Is some exception raised if I pass a probabilistic loss to those multivariate models?

Reply:
Yes, we raise an Exception, see [here](https://github.com/Nixtla/neuralforecast/blob/7aab005b8a2fa514a9178eb16d11dfbaf0fc71bd/neuralforecast/common/_base_multivariate.py#L91)